cpp/build-support/cpplint.py

```diff
@@ -4008,9 +4008,9 @@ def CheckTrailingSemicolon(filename, clean_lines, linenum, error):
 
   # Block bodies should not be followed by a semicolon.  Due to C++11
   # brace initialization, there are more places where semicolons are
-  # required than not, so we use a whitelist approach to check these
-  # rather than a blacklist.  These are the places where "};" should
-  # be replaced by just "}":
+  # required than not, so we use explicitly list the allowed rules
```

Comment:
This file is vendored; is it ok for us to edit it? (I'll note that https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py still says whitelist/blacklist)

Reply:
It's rarely updated so I think it's ok to edit it.